README.md

```diff
@@ -68,6 +68,35 @@ Some of the features that encourage more readable programs:
 - [ ] Trailing closure parameters can be passed outside the call parentheses.
 - [ ] Error propagation with `ErrorOr<T>` return type and dedicated `try` / `must` keywords.
 
+## Code reuse
+
+Jakt is flexible in how a project can be structured with a built-in module system.
```

Comment:
Should this also mention `import fn()`?

Reply:
I will have to play around with this, I am a little naive about `import fn()`.